docs/contents.md

```diff
@@ -42,10 +43,15 @@ When creating notification rules, selecting **Manual Input** as a notification t
 ## Playbook Collection
 
 | 02 - Use Case - FortiSOAR for Microsoft Teams |
-|:------------------------------------|
+|:----------------------------------------------|
+
+| Playbook Name                  | Description                                                                                                                    |
+|:-------------------------------|:-------------------------------------------------------------------------------------------------------------------------------|
+| Create Indicator               | Extract Indicators from the text provided as Input                                                                             |
+| Create Alert                   | Creates an alert based on the values provided in the form                                                                      |
```

Comment:
All three PBS are triggered from Ms teams as part of FSR command exec. We can have consistent description for them...